charts/operator/helm/README.md

```
@@ -30,13 +30,13 @@ Upgrading the version
 
 Please upgrade the CRDs first because Helm doesn't have a support to upgrade CRDs.
 
-    $ kubectl replace -f https://raw.githubusercontent.com/vdaas/vald/v1.7.17/charts/operator/helm/crds/valdrelease.yaml
-    $ kubectl replace -f https://raw.githubusercontent.com/vdaas/vald/v1.7.17/charts/operator/helm/crds/valdhelmoperatorrelease.yaml
+    $ kubectl replace -f https://raw.githubusercontent.com/vdaas/vald/v1.8.0/charts/operator/helm/crds/valdrelease.yaml
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)